db/migrate/20170912215933_create_orders.rb

```diff
@@ -2,6 +2,8 @@ class CreateOrders < ActiveRecord::Migration[5.1]
   def change
     create_table :orders do |t|
       t.integer :status
+      t.string :address
+      
```

Comment:
This is interesting. I thought we were going to create an address table with the `user_id` as the foreign key on that table? If so, we wouldn't need an address column here, right?

Reply:
This was the greatest point of indecision I had while working this. I ended up doing it this way because I felt that we needed to give it a space to hold a specific address. If we ended up doing order.user.addresses, we wouldn't know what the address associated with this particular order is. 

I know there has to be a better way of doing this, so I'll keep brainstorming this one!